packages/xo-web/src/common/intl/messages.js

```diff
@@ -967,6 +967,9 @@ const messages = {
   containerRestart: 'Restart this container',
 
   // ----- VM disk tab -----
+  attachDiskModalTitle: 'Attach disk',
+  attachDiskModalMessage:
+    "Are you sure you want to attach a disk on an illegal SR? The VM won't be able to boot or reboot.",
```

Comment:
`Incompatible` maybe better?

Reply:
Yes. Also, not "on" but maybe "from" or something like that.

And should we add a link to explain what makes it incompatible? As it may be hard to understand that it's not incompatible with the VM but only with the currently attached VDIs.

Reply:
Right :+1: